website/docs/contributing/reference/verify_a_release_candidate.md

```diff
@@ -89,55 +89,20 @@ Select `5` to trust the key ultimately.
 
 Now, we could start the verification. 
 
-Save the following content as `verify.sh`:
+We've provided a script to verify the checksum and signature of the release candidate. 
 
-```shell
-#!/bin/bash
-
-
-set -e
-
-YELLOW="\033[37;1m"
-GREEN="\033[32;1m"
-ENDCOLOR="\033[0m"
-
-if [ "$#" -ne 1 ]; then
-  echo "Usage: $0 {YOUR RELEASE TAR FILE}" >&2
-  exit 1
-fi
-
-PKG=$1
+The script is in the `scripts` directory of our repository.
+You can download it directly from [here](https://raw.githubusercontent.com/apache/incubator-opendal/main/scripts/check.sh)
+or check it out from the repository:
 
-if [ ! -f "$PKG" ]; then
-    echo "File '$PKG' does not exist."
-    exit 1
-fi
-
-echo "> Check signature"
-gpg --verify "$PKG.asc" "$PKG"
-
-if [ $? -eq 0 ]
-then
-    printf $GREEN"Success to verify the gpg sign"$ENDCOLOR"\n"
-else
-    printf $YELLOW"Failed to verify the gpg sign"$ENDCOLOR"\n"
-fi
-
-echo "> Check sha512sum"
-sha512sum --check "$PKG.sha512"
-
-if [ $? -eq 0 ]
-then
-    printf $GREEN"Success to verify the checksum"$ENDCOLOR"\n"
-else
-    printf $YELLOW"Failed to verify the checksum"$ENDCOLOR"\n"
-fi
+```shell
+git clone git@github.com:apache/incubator-opendal.git
 ```
 
-Run the following command to verify the checksum and signature:
+Run the script on a specific release candidate:
 
 ```shell
-./verify.sh apache-opendal-${release_version}-${rc_version}-src.tar.gz
+./check.sh apache-opendal-${release_version}-${rc_version}-src.tar.gz
```

Comment:
We need to use `./scripts/check.sh`

Reply:
I did not assume a specific location for running. 
It just said that this script needs to be executed in a specific way.

Reply:
> I did not assume a specific location for running.

No, all scripts must be run from the root dir of OpenDAL.

Reply:
![image](https://github.com/apache/incubator-opendal/assets/5351546/fd3d6da7-7a1b-479a-93db-996530514dcf)

Reply:
Updated.